packages/buckets/src/api/index.ts

```diff
@@ -426,112 +432,6 @@ export async function bucketsPushPath(
   return new Promise<PushPathResult>(async (resolve, reject) => {
     // Only process the first input if there are more than one
     const source: File | undefined = (await normaliseInput(input).next()).value
-    const client = grpc.client<
```

Comment:
This mess is basically due to removing the old function named bucketsPushPath, and replacing it with buckesPushPathNode... nothing really new here.